tests/models/onnxModels/model.onnxtxt

```diff
@@ -0,0 +1,133 @@
+ir_version: 3
+producer_name: "onnx-conv"
```

Comment:
Could you give a more descriptive name to the filename of the model?
E.g., simpleCNN.